edx_sga/templates/staff_graded_assignment/show.html

```diff
@@ -13,7 +13,7 @@
       <p>No file has been uploaded.</p>
     <% } %>
     <% if (graded) { %>
-      <p>Your score is <%= graded.score %> / <%= max_score %><br/>
+      <p>Your score is <%= (graded.score / max_score) * weight %> / <%= weight %><br/>
```

Comment:
@pdpinch A problem with weight of 0 is ungraded and does not show in the problem. See attached.
![prob_wt_1](https://cloud.githubusercontent.com/assets/899424/7991066/6fb7a344-0ac3-11e5-8a21-593cddb7b466.png)
![prob_wt_0](https://cloud.githubusercontent.com/assets/899424/7991065/6fb738dc-0ac3-11e5-9554-27b90cb5213e.png)